index.html

```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
+                        
+                    
+                </li>
+            </ul>
+                <button class="button-4">See Project</button>
+          
+        </div>
+        
+        </div>
+    </section>
+    <section>
+        <div class="container-1">
+            <a href="#" class="project-pics"><img class="project-pics" src="./images/Snapshoot Portfolio (1).png" alt=""></a>
+           <div class="flow">
+            <h3 class="project-title-1">Multi-Post <br> Stories</h3>
+            <ul class="info">
+                <li class="conopy-li">
+                    <h3 class="canopy">CANOPY</h3>
+                </li>
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li class="back-end-li">
+                    <h3 class="back-end">Back End Dev</h3>
+                </li>
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li>
+                    <h3 class="year">2015</h3>
+                </li>
+            </ul> 
+        <p class="info-about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="info-2">
+                <li>
+                    <button class="button-1">html</button>
+                        
+                </li>
+                <li>
+                    <button class="button-2">css</button>
+                        
+                    
+                </li>
+                <li>
+                    <button class="button-3">Javascript</button>
+                        
+                    
+                </li>
+            </ul>
+                <button class="button-4">See Project</button>
+          
+        </div>
+        
+        </div>
+    </section>
+    <section>
+        <div class="container-2">
+            <a href="#" class="project-pics"><img class="project-pics" src="./images/Snapshoot Portfolio (2).png" alt=""></a>
+           <div class="flow">
+            <h3 class="project-title">Tonic</h3>
+            <ul class="info">
+                <li class="conopy-li">
+                    <h3 class="canopy">CANOPY</h3>
+                </li>
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li class="back-end-li">
+                    <h3 class="back-end">Back End Dev</h3>
+                </li>
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li>
+                    <h3 class="year">2015</h3>
+                </li>
+            </ul> 
+        <p class="info-about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="info-2">
+                <li>
```

Comment:
I suggest you add equal `paddings and margins` to your project. Currently the left side has more one while the right does not. Its part of the requirements that your project matches figma design
<img width="1660" alt="Screenshot 2022-10-29 at 10 36 44" src="https://user-images.githubusercontent.com/48546992/198821326-5f17ec4e-7652-4a64-a5a4-8d2f57ef9449.png">

This is your design
<img width="359" alt="Screenshot 2022-10-29 at 11 18 44" src="https://user-images.githubusercontent.com/48546992/198821750-cb335374-75ae-49d1-a827-abc0279cc90a.png">

this is the figma design
<img width="381" alt="Screenshot 2022-10-29 at 11 18 51" src="https://user-images.githubusercontent.com/48546992/198821519-dec58a9f-04f0-4d84-9329-34f183947f40.png">

Reply:
![Screenshot 2022-10-29 183256](https://user-images.githubusercontent.com/110728909/198835499-27bb6769-14af-403c-a906-59cf70a0b86e.png)

-In my preview margins and paddings are set accordingly